tests/device_tests/test_msg_change_wipe_code_t2.py

```diff
@@ -26,7 +26,9 @@
 WIPE_CODE6 = "456789"
 WIPE_CODE_MAX = "".join(chr((i % 10) + ord("0")) for i in range(MAX_PIN_LENGTH))
 
-pytestmark = pytest.mark.skip_t1
+# Skipping TR just until we have the PIN entry - it causes
```

Comment:
These skips of `TR` are really necessary right now - it causes the `AssertionError: global buffer already used by another context` error in emulator, which is unrecoverable - the tests just stop and would time out in `CI`.

All other "normal" - recoverable - failing cases are let to fail freely.

As for forgetting - I have a [revert](https://github.com/trezor/trezor-firmware/pull/2233/commits/77fb75623613c77c4b7e140f8dd65a6d74f2f6f9) commit in the `PIN` branch.

Reply:
> the tests just stop and would time out in CI

that doesn't matter if the fixed code is coming in immediately afterwards. 

I'll go ahead and review the PIN PR so that we can merge them into one another and the problem is never seen in master. but please remove the skips from here, we don't need to create churn by introducing a commit that is immediately reverted

Reply:
Sure, I will delete both the "skipping" and "reverting" commits and force-push both branches, so the `PIN` PR shows a correct diff to `UI`